File: 75-SortColors/75-SortColors.js
// Last updated: 3/6/2026, 2:09:38 PM
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function (nums) {
    let temp;
    for (let i = 0; i < nums.length; i++) {
        for (let j = 0; j < nums.length - i - 1; j++) {
            if (nums[j] > nums[j + 1]) {
                temp = nums[j];
                nums[j] = nums[j+1];
                nums[j+1] = temp
            }
        }
    }
    return nums;
};

File: 75-SortColors/75-SortColors.cpp
// Last updated: 5/23/2026, 10:24:37 AM
1class Solution {
2public:
3    int findKthLargest(vector<int>& nums, int k) {
4        sort(nums.begin(), nums.end());
5        return nums[nums.size() - k];
6    }
7};